samples/cpp/benchmark_app/infer_request_wrap.hpp

```diff
@@ -59,7 +59,14 @@ class InferReqWrap final {
         _startTime = Time::now();
         _request.infer();
         _endTime = Time::now();
-        _callbackQueue(_id, _lat_group_id, get_execution_time_in_milliseconds());
+        double _execTime = getExecutionTimeInMilliseconds();
+        double difference = _targetLatency - _execTime;
```

Comment:
I suppose `_targetLatency` should be initialzied somehow to skip this code when command line parameter is not set